app/src/components/formdemo/FormDemo.tsx

```diff
@@ -749,6 +750,35 @@ export default function FormDemo() {
 		),
 	});
 
+	const yuri_data = `{
+	author: 'Yuri the Yeti',
+	gender: 'Male',
+	age: '33',
+	hobbies: ['sledding', 'ice fishing', 'building snow man'],
+	children: []
+}`;
+	demoSections.push({
+		title: 'Code Fence',
+		content: (
+			<Fragment>
+				<p>For code formatting</p>
+				<h3>Example</h3>
+				<pre>
+					<code>
+						{`<CodeFence 
+	title="Yuri data in json"
+	autoCollapse={false}
+	value={yuri_data}
+	language="json"
+/>`}
+					</code>
+				</pre>
+				<CodeFence title="Yuri data in json" autoCollapse={false} value={yuri_data} language="json" />
+				<CodeFence title="Yuri data in java" autoCollapse={false} value={yuri_data} language="java" />
```

Comment:
Use a separate value that's actually Java code (and C code for the next one, or change/delete it if you don't want to write C). It doesn't have to be much, but it needs to show the syntax highlighter is working by successfully parsing the value for that language.